circle.yml

```diff
@@ -31,23 +27,12 @@ test:
     - gvm use stable && go version
     - gvm use stable && make build:
         pwd: $BASE_STABLE
+    - gvm use stable && GO15VENDOREXPERIMENT=1 go get github.com/dgageot/docker-machine-driver-none-external
```

Comment:
@dgageot https://github.com/docker/docker-machine-driver-ci-test (for reference)